desktop-app/app/components/StatusBar/index.js

```diff
@@ -1,16 +1,55 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import cx from 'classnames';
 import styles from './styles.module.css';
 import Github from '../icons/Github';
 import {shell} from 'electron';
 import Twitter from '../icons/Twitter';
 import RoadMap from '../icons/RoadMap';
+import {ipcRenderer} from 'electron';
 
 const Spacer = ({width = 10}) => (
   <div className={styles.link} style={{width}}></div>
 );
 
+const AppUpdaterStatusInfo = ({status}) => {
+  let label = '';
+  switch(status) {
+    case 'checking':
+      label = 'Update Info: Checking for Updates...';
+      break;
+    case 'noUpdate':
+      label = 'Update Info: No Updates';
+      break;
+    case 'downloading':
+      label = 'Update Info: Downloading Update...';
+      break;
+    case 'downloaded':
+      label = 'Update Info: Update Downloaded';
+      break;
+    default:
+      label = null;
+    break;
+  }
+  if (label == null) return null;
+  return (
+    <span className={cx('appUpdaterStatusInfo', styles.linkText)}>
+      {label}
+    </span>
+  )
+}
+
 const StatusBar = () => {
+  const [appUpdaterStatus, setAppUpdaterStatus] = useState('idle');
```

Comment:
@jjavierdguezas Shall we move this appUpdator related logic into `AppUpdaterStatusInfo` component only? 
I feel, the StatusBar component doesn't need to know about it.

Reply:
done @manojVivek